### PR TITLE
CP-4712: remove deprecated logic, add legal note

### PR DESCRIPTION
**Pull request by KG-304**
image

### CODE REVIEW

File: mandrill/submittedapplication-staging.html [hunk 1]
```diff
@@ -1,1053 +1,1444 @@
...
+
+												<!--[if mso]>
 				<td valign="top" width="600" style="width:600px;">
 				<![endif]-->
-                <table align="left" border="0" cellpadding="0" cellspacing="0" style="max-width:100%; min-width:100%;" width="100%" class="mcnTextContentContainer">
-                    <tbody><tr>
-                        
-                        <td valign="top" class="mcnTextContent" style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;">
-                        
-                            <p dir="ltr">*|IF:COMPANY = New York & Company - Virginia|*
```
Comment (KG-304) [inline mandrill/submittedapplication-staging.html:634]: removed logic starting here

File: mandrill/submittedapplication-staging.html [hunk 2]
```diff
@@ -1,1053 +1,1444 @@
...
-
-                            *|ELSE:|*
```
Comment (KG-304) [inline mandrill/submittedapplication-staging.html:726]: ending here

File: mandrill/submittedapplication.html [hunk 3]
```diff
@@ -1,1053 +1,1444 @@
...
+
+												<!--[if mso]>
 				<td valign="top" width="600" style="width:600px;">
 				<![endif]-->
-                <table align="left" border="0" cellpadding="0" cellspacing="0" style="max-width:100%; min-width:100%;" width="100%" class="mcnTextContentContainer">
-                    <tbody><tr>
-                        
-                        <td valign="top" class="mcnTextContent" style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;">
-                        
-                            <p dir="ltr">*|IF:COMPANY = New York & Company - Virginia|*
```
Comment (KG-304) [inline mandrill/submittedapplication.html:634]: removed logic starting here

File: mandrill/submittedapplication.html [hunk 4]
```diff
@@ -1,1053 +1,1444 @@
...
-
-                            *|ELSE:|*
```
Comment (KG-304) [inline mandrill/submittedapplication.html:726]: ending here

File: mandrill/submittedapplication.html [hunk 5]
```diff
@@ -1,1053 +1,1444 @@
...
+																	<br>
+																	<br>
+																	<strong>Important Reminder</strong>:
+																	Please remember not to share sensitive information
+																	such as bank account details, social security
+																	numbers, or any other personal identification over
+																	email or SMS. Legitimate employers and recruitment
+																	agencies will never ask for this information via
+																	unsecured communication channels.
+																</p>
```
Comment (KG-304) [inline mandrill/submittedapplication.html:1023]: added legal text

**Review by gitsoufiane — APPROVED** [review]

**Comment by f1fe** [comment]
Thank you for adding these to version control

**Pull request merged**